mysql/parse_binary.go

```diff
@@ -19,15 +19,38 @@ func ParseBinaryUint16(data []byte) uint16 {
 	return binary.LittleEndian.Uint16(data)
 }
 
-func ParseBinaryInt24(data []byte) int32 {
+type Int24 int32
```

Comment:
Is this needed? Can we use the library function from here: https://github.com/go-mysql-org/go-mysql/blob/master/mysql/parse_binary.go#L22-L31